Makefile

```diff
@@ -19,6 +19,7 @@ all: clean fmt docs lint test
 setup: ## setup development dependencies
 	curl -L https://raw.githubusercontent.com/chanzuckerberg/bff/main/download.sh | sh
 	curl -s https://raw.githubusercontent.com/chanzuckerberg/terraform-provider-bless/main/download.sh | bash -s -- -b $(HOME)/.terraform.d/plugins -d
+	curl -s https://raw.githubusercontent.com/chanzuckerberg/terraform-provider-snowflake/main/download.sh | bash -s -- -b $(HOME)/.terraform.d/plugins -d
```

Comment:
should we pin a version?

Reply:
idk, I'd rather be testing against the latest version unless we want to specify a range of versions to test against

Reply:
otherwise we should change the provider constraints to a specific version?